### DISCUSSION TITLE
Making a package (ghcr.io) public has bad UI

### DISCUSSION THREAD

**Discussion by pkellner** [community/community#12965] [Actions]
Two things:

1) when you click "I Understand..." it does not give you any indication you clicked it"

2) if you forget to change the radio button to public, it should either warn you that you made no changes or not enable the button.

![image](https://user-images.githubusercontent.com/241170/158728447-21b3a1ec-0962-4e41-9519-b6da4cff49e4.png)

**Comment by pulkitaggarwl** [comment]
Thanks for the feedback, today we close the dialogue box once you click "I understand.. "
as well as post closing the setting suggests the visibility of package ex:  "This package is currently private" 

wanted to check if you noticed these indicators ? If these are not helping much will take this feedback and evaluate what can we do here.

**Reply by ericlery** [reply]
The problem seems solved, although the buttons still shouldn't be activated if we just write or paste the name of the repository/package. 

The actual behavior if we just write or paste the name of the repository is that the button will still activate, and we will not even have an alert/flash indicating that the package is already private, not really useful. For repository, we at least have the alert indicating that it's already private, although still not ideal.

The button must be disabled until we change the radio to public, and be disabled again if re-change to private, **or the modal should only be a toggle** : "Make it public / Make it private" in addition with the actual input requiring to enter the name of the repository
